main/templates/base.html

```diff
@@ -48,10 +48,10 @@
         if (typeof zE !== 'undefined') {
             zE('webWidget', 'prefill', {
                 name: {
-                    value: '{{user.name}}',
+                    value: '{% if user.is_authenticated %}{{ user.username }}{% endif %}',
```

Comment:
do you not think it should be `name` instead `username` ?

Reply:
I didn't find any `name` field in our `User` model.

Reply:
It is referring to user profile. please try with `user.profile.name`

Reply:
@asadiqbal08 Thanks for your review. I have incorporated the suggested change.